test/gemfiles/no-rails.Gemfile.lock

```diff
@@ -16,31 +16,37 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.7.0)
+    addressable (2.8.0)
       public_suffix (>= 2.0.2, < 5.0)
     crack (0.4.5)
       rexml
     diff-lcs (1.4.4)
     docile (1.4.0)
-    ethon (0.14.0)
+    ethon (0.15.0)
       ffi (>= 1.15.0)
     fabrication (2.22.0)
-    faraday (1.4.3)
+    faraday (1.8.0)
```

Comment:
Version 2 wasn't released yet when I created this PR. And since this gem already supports rails 6, it clearly works also with faraday 1, but I haven't looked into rails 7 yet and if there is a problem with rails 7 and faraday 1.x.

But I looked into if I should try to update to faraday 2 a few days ago and diaspora has other dependencies which still require faraday <2, so I would need to find a way to support both versions at the same time, because we can't use >=2 yet. But I haven't looked further into how easy it is to support both versions (because I thought faraday 2 has some breaking changes), and I'd probably prefer doing that in a separate PR.